sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -131,6 +201,18 @@
     "print(uri_folder_data_asset.path)"
```

Comment:
Could you please update the link to the github data.ipynb notebook to use relative paths in all the image notebooks. Please refer to "Check this notebook to create a compute cluster" link in the introduction of this notebook